uber/site_sections/preregistration.py

```diff
@@ -98,6 +98,48 @@ def index(self, message=''):
     def dealer_registration(self, message=''):
         return self.form(badge_type=c.PSEUDO_DEALER_BADGE, message=message)
 
+    def redeem(self, session, message='', **params):
+        accepted_message = 'Promo Code Applied!'
+        rejected_message = 'Promo Code Rejected!'
+        if 'promo' and 'edit_id' in params:
+            destroy_promo_code = False
+
+            attendee = cherrypy.session['unpaid_preregs'][params['edit_id']]
+            if attendee is not None:
+                pc = session.query(PromoCode).filter(PromoCode.code == params['promo']).first()
```

Comment:
In general, unless a variable is 'temporary' (e.g., only used in a lamba), description > brevity. So `promo` or `promo_code` would be better than `pc` here. Otherwise, someone reading the code will have to search through 20+ lines of code for the meaning of `pc` (which I just had to do). Same for pcu - it's totally okay to just call it `promo_code_usage` or `promo_usage`.